### DISCUSSION TITLE
Where did my ZFS pool go on reboot

### DISCUSSION THREAD

**Discussion by Mokerak** [openzfsonwindows/openzfs#311] [Q&A] [answered]
Kinda worried, where did my ZFS pool go when I rebooted my computer.

When running 'zpool.exe list' it says there are no pools available, and 'zfs list' says no datasets available.

Is there some command I should be running each time I boot up???
How do I remount it?

**Comment by EchterAgo** [comment, marked as answer]
You need to import the pool: https://github.com/openzfsonwindows/openzfs/tree/windows/module/os/windows#importing-a-pool

There are ways to automate this but I've never done it.

**Reply by Mokerak** [reply]
Thanks a bunch!

Would it be advisable to set up a script to just run this command? Or is there a better way through ZFS itself? I know how to just set up like a .bat script to run in windows so that should be easy

**Reply by EchterAgo** [reply]
FYI, this is how others automated the zpool import: https://openzfsonosx.org/wiki/Windows_import

Beware that there might be conditions where your system will BSOD on boot/import, see https://openzfsonosx.org/wiki/Windows_bootloop
